stable/neo4j/templates/core-statefulset.yaml

```diff
@@ -1,8 +1,9 @@
-apiVersion: "apps/v1"
+apiVersion: "apps/v1beta2"
```

Comment:
I believe this breaks compatibility with `k8s 1.16+` - see https://github.com/helm/charts/pull/20942.

Running `helm install my-neo4j stable/neo4j --set acceptLicenseAgreement=yes --set neo4jPassword=mySecretPassword` against `k8s 1.16+` yields ```Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "StatefulSet" in version "apps/v1beta2"```